src/Sale/lib.mo

```diff
@@ -397,6 +400,9 @@ module {
 
     // getters & setters
     public func availableTokens() : Nat {
+      if (Env.openEdition) {
```

Comment:
should we return a specific flag that tells the frontends that this is an open edition mint? or do you think the combination of `availableTokens` of `1` and a specified `endTime` is enough already?